src/opm/parser/eclipse/share/keywords/000_Eclipse100/P/PVTWSALT

```diff
@@ -0,0 +1,9 @@
+{"name" : "PVTWSALT" ,
+"sections" : ["PROPS"],
+"size" : "UNKNOWN",
+"records": [
+[{"name":"P_REF", "value_type" : "DOUBLE", "dimension":"Pressure" },
```

Comment:
Ahhh - this was *really* unlucky; you have now encountered a keyword structure which we do not really support - alternating record structure - we will have to look into this. I don't know - but I do fear that your implementation works by chance.

Reply:
We noticed while we worked with implementing PVTWSALT. The current implementation just reads the keyword naively and trust the user to handle the data correctly. It is tested, so it should work, even thought it is not the nicest piece of code.

Reply:
Since #1326 is merged. Does this mean that the usage of PVTWSALT in this PR needs to changed?